defaults/config.js

```diff
@@ -1,510 +1,428 @@
-	// Set to 'false' to enable users.
+	// When set to `true`, The Lounge starts in public mode. When set to `false`,
+	// it starts in private mode.
 	//
-	// @type     boolean
-	// @default  false
+	// - A **public server** does not require authentication. Anyone can connect
+	//   to IRC networks in this mode. All IRC connections and channel
+	//   scrollbacks are lost when a user leaves the client.
+	// - A **private server** requires users to log in. Their IRC connections are
+	//   kept even when they are not using or logged in to the client. All joined
+	//   channels and scrollbacks are available when they come back.
 	//
+	// This value is set to `false` by default.
 	public: false,
 
+	// ### `host`
 	//
-	// IP address or hostname for the web server to listen on.
-	// Setting this to undefined will listen on all interfaces.
-	//
-	// For UNIX domain sockets, use unix:/absolute/path/to/file.sock.
+	// IP address or hostname for the web server to listen to. For example, set it
+	// to `"127.0.0.1"` to accept connections from localhost only.
 	//
-	// @type     string
-	// @default  undefined
+	// For UNIX domain sockets, use `"unix:/absolute/path/to/file.sock"`.
 	//
+	// This value is set to `undefined` by default to listen on all interfaces.
 	host: undefined,
 
+	// ### `port`
 	//
-	// Set the port to listen on.
-	//
-	// @type     int
-	// @default  9000
+	// Set the port to listen to.
 	//
+	// This value is set to `9000` by default.
 	port: 9000,
 
+	// ### `bind`
 	//
-	// Set the local IP to bind to for outgoing connections. Leave to undefined
-	// to let the operating system pick its preferred one.
-	//
-	// @type     string
-	// @default  undefined
+	// Set the local IP to bind to for outgoing connections.
 	//
+	// This value is set to `undefined` by default to let the operating system
+	// pick its preferred one.
 	bind: undefined,
 
+	// ### `reverseProxy`
 	//
-	// Sets whether the server is behind a reverse proxy and should honor the
-	// X-Forwarded-For header or not.
-	//
-	// @type     boolean
-	// @default  false
+	// When set to `true`, The Lounge is marked as served behind a reverse proxy
+	// and will honor the `X-Forwarded-For` header.
 	//
+	// This value is set to `false` by default.
 	reverseProxy: false,
 
+	// ### `maxHistory`
 	//
-	// Set the default theme.
-	// Find out how to add new themes at https://thelounge.github.io/docs/plugins/themes.html
+	// Defines the maximum number of history lines that will be kept in memory per
+	// channel/query, in order to reduce the memory usage of the server. Setting
+	// this to `-1` will keep unlimited amount.
 	//
-	// @type     string
-	// @default  "default"
-	//
-	theme: "default",
+	// This value is set to `10000` by default.
+	maxHistory: 10000,
 
+	// ### `https`
 	//
-	// Prefetch URLs
+	// These three settings are used to run The Lounge using encrypted HTTP/2 on
+	// the server side. This will fallback to regular HTTPS if HTTP/2 is not
+	// supported.
 	//
```

Comment:
Add a line saying reverse proxy is preferred if you want more options / live reload of certs

Reply:
Will do.

Reply:
Done, linked to the [upcoming guide on reverse proxies](https://5acafe274ed62f0a99ee7c04--thelounge-chat.netlify.com/docs/guides/reverse-proxies.html).